File: dec2bin.py
from time import time_ns

n=int(input("Give me a number: "))
start=time_ns()
bin=""

if n==0:
    print(0)

while n>0:
    bin+=f"{n%2}"
    n//=2
    
print(bin[::-1])
print(f"\nTime taken: {time_ns()-start}ns.")

File: dec2bin.java
import javax.swing.*;

class protochess{
    public static void main(String Args[]){
        //Max int Value = 2147483647
        int n = Integer.parseInt(JOptionPane.showInputDialog("Give me a Number: "));;
        String bin = "";
        long start = System.nanoTime();


        while(n>0){
            bin+=Integer.toString(n%2);
            n/=2;
        }

        char[] reverser = bin.toCharArray();
        char temp;
        
        for(int i=0; i<bin.length();i++){
            for(int j=0; j<bin.length()-1-i;j++){
                temp = reverser[j+1];
                reverser[j+1] = reverser[j];
                reverser[j] = temp;
            }
        }
        
        bin = new String(reverser);

        JOptionPane.showMessageDialog(null, "Your value is: "+bin+"\n Time elapsed: "+(System.nanoTime()-start)+"ns.");
    }
}